README.md

```diff
@@ -172,6 +172,10 @@ else
 fi
 ```
 
+You can completely omit the `docker/build` script if you don't need any extra
```

Comment:
I messed up a commit, a readme that should have been amended in a commit was done in the wrong one. I will fix the commits.